src/js/keyboardInput.js

```diff
@@ -285,6 +291,13 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 method: "on",
                 args: ["focusin.speakOnFocusMessage", "{that}.speakOnFocusMessage"],
                 priority: 1
+            },
+            // The shiftLatchChange.speakShiftState listener has
+            // a priority of "last" as it must happen after the model
+            // is changed via the shiftLatchChange.toggleShiftLatched listener
+            "shiftLatchChange.speakShiftState": {
+                listener: "{that}.speakShiftState",
+                priority: "last"
```

Comment:
A fixed priority rather than "last" is better here - since this could well cause problems in test cases etc. with the framework currently in use